js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -2985,4 +2985,16 @@ readBlog=Read blog
 customValue=Custom value
 termsAndConditionsUserAttribute=Terms and conditions accepted timestamp
 realmOverridesDescription= Realm overrides allow you to specify translations that will take effect for the entire realm. These translations will override any translation specified by a theme.
-addTranslation=Add translation
\ No newline at end of file
+addTranslation=Add translation
+clientsClientScopesHelp=The scopes associated with this resource.
+searchItem=Search item
+createItem=Create item
+itemDelete=Delete item
+itemDeleteConfirm=Are you sure you want to permanently delete the item
+itemDeleteConfirmTitle=Delete item?
+itemDeletedSuccess=The item has been deleted
+itemDeleteError=Could not delete item: {{error}}
+noItems=There are no items
+noItemsInstructions=You haven't created any items in this realm. Create a item to get started.
+itemSaveError=Error could not save item\! {{error}}
+itemSaveSuccessful=Sucessful saved
```

Comment:
I wouldn't change this for this first version, but in the future a developer would want to use the actual name of the item.

So for instance, `searchItem=Search item` would become `searchItem=Search {{itemName}}`